ui/app/templates/components/dashboard/learn-more-card.hbs

```diff
@@ -11,13 +11,12 @@
           (not learnMoreLink.requiredFeature)
         )
       }}
-        <Hds::Link::Standalone
-          @icon={{learnMoreLink.icon}}
-          @text={{learnMoreLink.title}}
-          @href={{learnMoreLink.link}}
-          class="has-bottom-margin-xs"
-          data-test-learn-more-link={{learnMoreLink.title}}
-        />
+        <div class="has-bottom-margin-xxs">
+          <DocLink data-test-learn-more-link={{learnMoreLink.title}} @path={{learnMoreLink.link}}>
```

Comment:
DocLink should support `...attributes` - curious why the extra div was necessary 🤔

Reply:
Removed the div in https://github.com/hashicorp/vault/pull/21057/commits/0f235dd562f68d26296d5b8627cd7ea1df060f6c